### PR TITLE
Simplify / Clarify README

### PR DESCRIPTION
**Pull request by lazaridiscom**
- rearrange sections
- CAPITALS in titles
- and more...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,80 +13,73 @@
...
-----
-
-[Concord-BFT Slack](https://concordbft.slack.com/).
-
-Get Slack invitation via this [link](https://join.slack.com/t/concordbft/shared_invite/enQtNTAyMzg0MTk0NzU0LWViMjA4ODQ4OTcxYmVkMWU4NGYyNTZmZDQ0ZTIzZTNiZmIyZmI5Mjg0NDZjNjBkYmI2Yjg1Mzg1NGUyNjQ0OWQ) or send request to <concordbft@gmail.com>.
-
+## Build (Ubuntu Linux 18.04)
 
+We use Conan Package Manager to install all concord-bft dependencies.
+Dependencies that currently are not supported by the conan center, have custom conan installer in concord-bft/.conan
```
Comment (teoparvanov) [inline README.md]: grammar/typos:
- "...that are currently not supported..."
- "...have a custom..."

File: README.md [hunk 2]
```diff
@@ -13,80 +13,73 @@
...
-We use Conan Package Manager to install all concord-bft dependencies.
-Dependencies that currently are not supported by the conan center, have custom conan installer in concord-bft/.conan
+### Build Options
 
-### Install Dependencies
-Install all dependencies using
+In order to turn on or off various options, you need to change your cmake configuration. This is
+done by passing arguments to cmake with a `-D` prefix: e.g. `cmake -DBUILD_TESTING=OFF`. Note that
+make must be run afterwards to build according to the configuration. The following options are
+useful for building concord-bft:
```
Comment (teoparvanov) [inline README.md]: I'd rather say: "The following options are available:"

File: README.md (hunk 1 above)
Comment (lazaridiscom) [reply README.md]: that's not my text, the diff falsely reports as new.

This needs further iterations anyways, needs to become even simpler.

File: README.md [hunk 3]
```diff
@@ -172,3 +136,37 @@ Run the following from the top level concord-bft directory:
...
+signed our contributor license agreement (CLA), our bot will update the issue when you open a Pull Request. For any
+questions about the CLA process, please refer to our [FAQ](https://cla.vmware.com/faq). For more detailed information,
+refer to [CONTRIBUTING.md](CONTRIBUTING.md).
+
+## Community
+
+
+[Concord-BFT Slack](https://concordbft.slack.com/).
+
+Get Slack invitation via this [link](https://join.slack.com/t/concordbft/shared_invite/enQtNTAyMzg0MTk0NzU0LWViMjA4ODQ4OTcxYmVkMWU4NGYyNTZmZDQ0ZTIzZTNiZmIyZmI5Mjg0NDZjNjBkYmI2Yjg1Mzg1NGUyNjQ0OWQ) or send request to <concordbft@gmail.com>.
```
Comment (teoparvanov) [inline README.md]: This link doesn't seem to work anymore. Invitation links usually expire - we should provide a more persistent entry point here.

Comment (teoparvanov) [reply README.md]: @salieri11 , what would you suggest?

File: README.md (hunk 2 above)
Comment (lazaridiscom) [reply README.md]: again not my text, the diff got confused.

**Comment by lazaridiscom** [comment]
(i used the github ui, if you can squash on your side or don't care, would be fine. otherwise i can squash in one or two days from a machine setup).

**Comment by teoparvanov** [comment]
> 
> 
> (i used the github ui, if you can squash on your side or don't care, would be fine. otherwise i can squash in one or two days from a machine setup).

We will squash through the UI, don't bother squashing locally, as it would re-trigger the CI run.
Let's wait for at least another team member to chime in on this PR, before we finalize it.

**Pull request merged**